src/AccretionFormulae.jl

```diff
@@ -1,8 +1,7 @@
 module AccretionFormulae
 
-import GeodesicBase
 import GeodesicBase: AbstractMetricParams
-import CarterBoyerLindquist: CarterMethodBL, Σ, Δ, A, rms, Σδr_δλ
+import CarterBoyerLindquist: CarterMethodBL, Σ, Δ, A
```

Comment:
Imports are being removed here that are needed (for redshift code).

Reply:
I think this was accidental, but I have noticed that the code doesn't work with this version (there'll be more detail in the PR I'm about to send).